scalyr

```diff
@@ -559,55 +557,27 @@ def commandTimeseriesQuery(parser):
     # Get the API token.
     apiToken = getApiToken(args, 'scalyr_readlog_token', 'Read Logs')
 
-		# build the query, in either flavor
+		# build the query
     query = {
         "queryType": "numeric",
+        "filter": args.filter[0],
+        "function": args.function,
```

Comment:
Since we're only picking off the immediate arg.

Reply:
I don't follow?  Again this just replicates `commandNumericQuery`.  Not quite sure why those are accessed as array types with `nargs=1`, but I'm not a python guy =)

Reply:
In other words, I would expect this to just be:

```
"filter": args.filter,
```

Without the `[0]`, but it does work as-is and has been that way forever in `commandNumericQuery`, so I'm chalking this up to my unfamiliarity with the language in general and `argparse` in particular.

Reply:
No worries. I missed the first point about it being an optional arg, hence the `*` use. Apparently, the args are returned as a list/tuple, hence the `[0]` indexing.

Oddly enough, Python offers many ways to do cmdline arg parsing.